src/applications/mhv-medical-records/components/LabsAndTests/MicroDetails.jsx

```diff
@@ -125,16 +125,12 @@ ${record.results}`;
 
       <div className="test-details-container max-80">
         <h2>Details about this test</h2>
-        {record.labType && (
-          <>
-            <h3 className="vads-u-font-size--md vads-u-font-family--sans">
-              Lab type
-            </h3>
-            <p data-testid="microbio-sample-tested" data-dd-privacy="mask">
-              {record.labType}
-            </p>
-          </>
-        )}
+        <h3 className="vads-u-font-size--md vads-u-font-family--sans">
+          Date and time collected
+        </h3>
+        <p data-testid="microbio-date-completed" data-dd-privacy="mask">
+          {record.date}
+        </p>
```

Comment:
This field is duplicated exactly in the `DateSubheading` above.

Reply:
Good catch!